pipeline/src/pipeline/do_truthdata.jl

```diff
@@ -16,3 +16,22 @@ function do_truthdata(pipeline::AbstractEpiAwarePipeline)
     end
     return truthdata_from_configs
 end
+
+"""
+Generate truth data for the EpiAwarePipeline. This is example mode.
+
+# Arguments
+- `pipeline::EpiAwareExamplePipeline`: The EpiAwarePipeline object.
+
+# Returns
+An array of truth data generated from the given pipeline.
+
+"""
+function do_truthdata(pipeline::EpiAwareExamplePipeline)
```

Comment:
Same point here about relying on the default

Reply:
Doesn't this really suggest a new method for make truth data configs?

Reply:
As per #289 ? That makes sense